quickstart-docs: Add ssh-agent instructions (#325)
kinvolk/lokomotive: docs/quickstarts/aws.md

### Step 5: Create Lokomotive cluster

> Lokoctl uses Terraform behind the scene and it needs SSH access to controller nodes. For SSH to work add at least one SSH private key corresponding to the public keys specified earlier, to your `ssh-agent`.

Reviewer: ```suggestion
Add a private key corresponding to one of the public keys specified in `ssh_pubkeys` to your ssh-agent:
```

- IMO it's irrelevant to mention that lokoctl uses Terraform here. All the user needs to know is that they should add an SSH key to ssh-agent.
- "Lokoctl" should be `lokoctl`.
- "behind the scene" should be "behind the scenes".
- I like using a colon at the end of an instruction that is followed by a code snippet. We are currently inconsistent there but if we should settle on something, I suggest it be a colon rather than a period.

> Lokoctl uses Terraform behind the scene and it needs SSH access to controller node. For SSH to work add SSH private key corresponding to the public keys specified earlier. Add your SSH private key to `ssh-agent`.

Reviewer 2: ```suggestion
Lokoctl uses Terraform behind the scene and it needs SSH access to controller nodes. For SSH to work add at least one SSH private key corresponding to the public keys specified earlier to your `ssh-agent`.
```

Author: Spelling it controller node is intentional. Since it needs access to just one node, the zeroth one. After that bootkube does it's magic, isn't it how it works?

Reviewer 2: We need SSH access to all controller nodes, to copy secrets: https://github.com/kinvolk/lokomotive/blob/master/assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/ssh.tf#L3
